docs/api/emailauth.md

```diff
@@ -14,12 +14,6 @@ To create an emailauth for the current user:
 POST /user/emailauths HTTP/1.1
 ```
 
-To create an emailauth for another user:
-
-```http
-POST /users/:user/emailauths HTTP/1.1
```

Comment:
This does not work since you are deleting it?

Reply:
I couldn't find the url handler in the code that is supposed to do this, so that's why i deleted it.

Reply:
is/was that used in any its-tools script?

Reply:
not by my knowledge.. if I was to update it for another user, I would try impersonating first. So if you are sure, go ahead and merge 👍